plugins/linux/ifplugin/ifaceidx/ifaceidx.go

```diff
@@ -68,6 +68,8 @@ type LinuxIfMetadata struct {
 	VPPTapName   string // empty for VETHs
 	Namespace    *linux_namespace.NetNamespace
 	HostIfName   string
+	Vrf          string
+	VrfDevRT     int // only set for VRF_DEVICE
```

Comment:
Shouldn't this be `uint32` or at least `uint`?

Reply:
Agree that it should be and will change it.
I used `int` only because netlink package also uses it for the table ID, but negative values would not be valid, so...

Reply:
Right. I see they use `int`, but only in [`Route`](https://pkg.go.dev/github.com/vishvananda/netlink#Route). In the [`Vrf`](https://pkg.go.dev/github.com/vishvananda/netlink#Vrf) it uses `uint32`.